script/brew-publish

```diff
@@ -46,7 +46,7 @@ rm -f "${formula}.bak"
 git commit -m "${brew_name} ${version#v}" -- "$formula"
 # hackish way of getting the git remote name for user's fork
 fork_remote="$(hub fork 2>&1 | grep -oE 'remote:? \S+' | tail -1 | awk '{print $2}')"
-git push -u "$fork_remote" "$branch"
+git push -u "${fork_remote}" HEAD
```

Comment:
consistency with https://github.com/rbenv/ruby-build/pull/877/files#diff-b746614a124d886e7bcb47a43fce3a17R46 and https://github.com/rbenv/ruby-build/pull/877/files#diff-b746614a124d886e7bcb47a43fce3a17R50

Can remove if desired